tests/unit/users/test_serializers.py

```diff
@@ -7,6 +7,7 @@
 from rest_framework.test import APIRequestFactory
 
 
+@patch('openslides.users.rest_auth.config', {'system_enable_anonymous': False})
```

Comment:
I don't know, but it does not work without the patch. Probably because it is a unittest and the database is not loaded. If you like you can try to fix the config, so it works without a database (I would like it)